src/css/profile/mobile/common/datepicker.less

```diff
@@ -0,0 +1,57 @@
+.ui-date-picker {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+
+  &-header {
+    .font(regular);
+    display: flex;
+    width: 100%;
+    height: 36 * @px_base;
+    font-size: 17 * @sp_base;
+    color: var(--date-picker-header-text-color);
+    line-height: 36 * @px_base;
+    align-items: center;
+    justify-content: center;
+    @media (orientation: landscape) {
```

Comment:
In the Guide , is it divide according to orientation mode??

Reply:
You are right. I checked another page.